plumbing/transport/http/common.go

```diff
@@ -245,6 +263,8 @@ func newSession(c *client, ep *transport.Endpoint, auth transport.AuthMethod) (*
 			configureTransport(transport, ep)
```

Comment:
Is the error swallowing here on purpose? It seems like this could result in unexpected behavior.

Reply:
I don't think so, we should at the very least log it. Do you mind handling the errors from the `configureTransport` calls into a separate commit please?